books/free-programming-books-langs.md

```diff
@@ -291,6 +291,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [Getting Started with Arduino products](https://www.arduino.cc/en/Guide) - Official Arduino Documentation (:construction: *in process*)
 * [Introduction to Arduino](http://playground.arduino.cc/Main/ManualsAndCurriculum)
 * [Introduction to Arduino : A piece of cake!](http://www.introtoarduino.com) - Alan G. Smith
+* [Learn Arduino](https://riptutorial.com/Download/arduino.pdf) - riptutorial.com (PDF)
```

Comment:
Please add the "Compiled from StackOverflow documentation" attribution we use for goalkicker and riptutorial books